PocketKit/Sources/Textile/Style/PocketButtonStyle.swift

```diff
@@ -57,73 +66,90 @@ public struct PocketButtonStyle: ButtonStyle {
         }
     }
 
-    public init(_ variation: Variation) {
+    public init(_ variation: Variation, _ size: Size = .normal) {
         self.variation = variation
+        self.size = size
     }
 
     public func makeBody(configuration: Configuration) -> some View {
-        PocketButton(configuration: configuration, variation: self.variation)
+        PocketButton(configuration: configuration, variation: self.variation, size: self.size)
     }
 
     struct PocketButton: View {
         let configuration: ButtonStyle.Configuration
 
         let variation: Variation
+        let size: Size
 
         @Environment(\.isEnabled)
         private var isEnabled: Bool
 
         var body: some View {
-            HStack {
-                Spacer()
+            // TODO: Refactor this to match our components in Figma
+            if size == .small && variation == .primary {
                 configuration.label
```

Comment:
Consider pulling each block into it's own function.

Reply:
I think we should refactor this entirely instead so didn't want to make too many modifications on existing code